sqlserver/datadog_checks/sqlserver/sqlserver.py

```diff
@@ -205,6 +208,49 @@ def _new_query_executor(self, queries):
     def set_resolved_hostname_metadata(self):
         self.set_metadata('resolved_hostname', self.resolved_hostname)
 
+    def set_resource_tags(self, resolved_hostname):
+        if self.cloud_metadata.get("gcp") is not None:
+            self.tags.append(
+                "dd.internal.resource:gcp_sql_database_instance:{}:{}".format(
+                    self.cloud_metadata.get("gcp")["project_id"],
+                    self.cloud_metadata.get("gcp")["instance_id"]
+                )
+            )
+        if self.cloud_metadata.get("aws") is not None:
+            self.tags.append(
+                "dd.internal.resource:aws_rds_instance:{}".format(
+                    self.cloud_metadata.get("aws")["instance_endpoint"],
+                )
+            )
+        elif AWS_RDS_HOSTNAME_SUFFIX in self.hostname:
```

Comment:
Should this be `resolved_hostname` here instead of `self.hostname`?

Reply:
ah yes, thanks